.github/workflows/symfony-console.yml

```diff
@@ -10,6 +10,9 @@ concurrency:
   cancel-in-progress: true
 jobs:
   symfony-console:
+    strategy:
+      matrix:
+        app-id: [ admin, oauth-api, front ]
```

Comment:
The name "oauth-api" might mislead developers or users into thinking it's the standard OAuth implementation, whereas it's just an API utilizing OAuth. This could lead to misunderstandings or incorrect expectations regarding the actual functionality of the API.

By naming the API based on a specific technology like OAuth, you might restrict its scope. (The name "oauth-api" doesn't clearly describe the API's primary purpose. Users might struggle to understand exactly what the API does without looking into its detailed documentation)

Reply:
And if the API evolves to use other authentication or authorization methods in the future, the name "oauth-api" could become obsolete or misleading.

Reply:
@M0rgan01 

On the other hand, we already have domain.com/api for web services. We could name it `modern-api` or `new-api` but we know it could potentially end up being "not that modern" or "not that new" API.

Reply:
and `api/v2` ?

Reply:
I don't like v2 because it implies there was v1 of this new oauth API in a way, whereas it's just in opposition to the webservice API which is not really a v1 🤔

But I understand your point, why not `backoffice-api` then? or `bo-api` or `admin-api` It does represent the purpose of this new API which is only to target the BO so far

And when we'll introduce a new modern API for the front office we can follow the same convention of `front-api` or `fo-api` Changes are that the two APIs will be completely different in their contract, they will also probably have different clients to authenticate, different permissions and so on, so it makes sense to split them apart completely maybe?

Reply:
I like the idea, `bo-api` / `backoffice-api` :+1:

Reply:
So after the discussion we had with other devs I modified the base path with `admin-api` 
And to keep things consistent the dedicated kernel is now `AdminAPIKernel` which is more in line with `AdminKernel` and `FrontKernel` That's what made us lean towards the `admin-api` naming